src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/JdbcTestIT.java

```diff
@@ -78,4 +80,50 @@ public void testGroupByInQuery() {
     private JSONObject executeJdbcRequest(String query){
         return new JSONObject(executeQuery(query, "jdbc"));
     }
+
+    @Test
+    public void numberOperatorNameCaseInsensitiveTest() {
+        assertEquals(
+                executeQuery("SELECT ABS(age) FROM elasticsearch-sql_test_index_account/account " +
+                                "WHERE age IS NOT NULL LIMIT 5", "jdbc"),
+                executeQuery("SELECT abs(age) FROM elasticsearch-sql_test_index_account/account " +
+                                "WHERE age IS NOT NULL LIMIT 5", "jdbc")
+        );
+    }
+
+    @Test
+    public void trigFunctionNameCaseInsensitiveTest() {
+        assertEquals(
+                executeQuery("SELECT Cos(age) FROM elasticsearch-sql_test_index_account/account " +
+                                "WHERE age is NOT NULL LIMIT 5", "jdbc"),
+                executeQuery("SELECT cos(age) FROM elasticsearch-sql_test_index_account/account " +
+                                "WHERE age is NOT NULL LIMIT 5", "jdbc")
+        );
+    }
+
+    @Test
+    public void stringOperatorNameCaseInsensitiveTest() {
+        assertEquals(
+                executeQuery(
+                        "SELECT SubStrinG(lastname, 0, 2) FROM elasticsearch-sql_test_index_account/account " +
+                                "ORDER BY age LIMIT 5", "jdbc"),
+                executeQuery(
+                        "SELECT substring(lastname, 0, 2) FROM elasticsearch-sql_test_index_account/account " +
+                                "ORDER BY age LIMIT 5", "jdbc")
+        );
+    }
+
+    @Test
+    public void dateFunctionNameCaseInsensitiveTest() {
+        assertEquals(
+                executeQuery("SELECT insert_time FROM elasticsearch-sql_test_index_online/online " +
```

Comment:
`insert_time` should be changed to what is present in `GROUP BY` (`date_format(...)`)

Reply:
Field name inconsistent with the aggregation again. Sorry and will fix it. Thanks!